app/src/layouts/kanban/kanban.vue

```diff
@@ -185,6 +196,13 @@ function saveChanges() {
 	editDialogOpen.value = null;
 	editTitle.value = '';
 }
+
+// Used to find relevant information to display cards' formatted text with <render-display>
+const textFieldConfiguration = computed<Field | undefined>(() => {
+	/* Extra information is stored in fieldsInCollection,
+	but we need to find the config for the field used as the card's text field */
+	return props.fieldsInCollection.find((field) => field.field === props.layoutOptions.textField);
+});
```

Comment:
I contrast to the Tabular layout, I could not find a pretty declaration of the settings / configurations for the fields that are used for the kanban card's title, text, tags etc.

By settings / configuration I mean the definition of the item's display configuration, type of field (e.g. text / string) and so on.

Instead, I am getting the configuration for the field that is selected to be the kanban card's text by looking for the field that is selected to be the card's text in the given collection's array of field configurations. In cases where no field has been selected for the card's text, this will return `undefined` and the `<render-display>` will not render on the card, just like before.

Let me know if you know of an easier / better way of getting these data without having to look them up manually. This way seems robust enough and the computational overhead is extremely small, even if this method is applied to other card fields in the future where the configuration is also needed (such as the card labels etc), since it will only ever need compute this once for every field on the collection (not every item being displayed).

Reply:
This looks good to me! The layoutOptions do in fact just store the string field name, so you will have to lookup the field information from there.